detr_models/detr/matcher.py

```
@@ -96,10 +95,10 @@ def prepare_cost_matrix(
         Note that the number of objects refers to the objects inside the batch. For the bipartite assignment,
         these get sliced to match only the considered sample.
     """
-
     config = DefaultDETRConfig()
```

Still working with default configuration, so it is not possible to use custom configurations!

---

Yes, that needs to be done in another PR. Here, we just wanted to fix the parameterization. The idea at the beginning was to include a config file which can be manipulated by everybody depending the use-case. I totally agree that the naming (**Default**Config) is probably misleading.